packages/client-sdk-web/src/utils/auth.ts

```diff
@@ -0,0 +1,4 @@
+import {InternalSuperUserPermissions} from '@gomomento/sdk-core/dist/src/internal/utils/auth';
+import {AllDataReadWrite} from '@gomomento/sdk-core';
```

Comment:
lol I'm sorry... AllDataReadWrite is indeed public.  It's just the `Internal` one that is not.  but otherwise this looks good